lib/msal-angular/docs/multi-tenant.md

```diff
@@ -45,3 +45,42 @@ export class AppComponent implements OnInit {
     });
   }
 ```
+
+## Multi-tenant access tokens
+
+By default the MsalInterceptor will retrieve access tokens from the users' primary tenant. You can however override this behavior by setting a dynamic auth request on the MsalInterceptorConfig.
+
+```js
+export function MSALInterceptorConfigFactory(): MsalInterceptorConfiguration {
+  const protectedResourceMap = new Map<string, Array<string>>();
+  protectedResourceMap.set("https://graph.microsoft.com/v1.0/me", ["user.read"]);
+  
+  return {
+    interactionType: InteractionType.Popup,
+    protectedResourceMap,
+    authRequest: {
+      authority: 'https://login.microsoftonline.com/organizations',
+    },
+    dynamicAuthRequest: (authRequest, httpReq) => {
+      return {
+        ...authRequest,
+        authority: `https://login.microsoftonline.com/${authRequest.account?.tenantId ?? 'organizations'}`
```

Comment:
Instead of pull this off of the authRequest, you should use the get account APIs on the MSAL instance (either getActiveAccount, getAllAccounts[0], etc) depending on if your app sets the active account.